src/Umbraco.Web.BackOffice/Controllers/BackOfficeController.cs

```diff
@@ -329,7 +329,7 @@ public async Task<JavaScriptResult> ServerVariables()
     [AllowAnonymous]
     public ActionResult ExternalLogin(string provider, string? redirectUrl = null)
     {
-        if (redirectUrl == null || Uri.TryCreate(redirectUrl, UriKind.Absolute, out _))
+        if (redirectUrl == null || !redirectUrl.StartsWith("/"))
```

Comment:
This was a bit of a tricky one. 😅
I was trying to reproduce it in my Windows laptop and it took me a while to realize that the issue was only present in Linux/macOS.
Found this dotnet issue about it: https://github.com/dotnet/runtime/issues/22718
```suggestion
        // Only relative urls are accepted as redirect url.
        // We can't simply use Uri.TryCreate with kind Absolute, as in Linux any relative url would be seen as an absolute file uri
        if (redirectUrl == null || !Uri.TryCreate(redirectUrl, UriKind.RelativeOrAbsolute, out Uri? redirectUri) || redirectUri.IsAbsoluteUri)
```
^ I do believe that this suggestion is better, but let me know what you think! 🙂